plugin/src/sidebar.ts

```diff
@@ -394,7 +410,16 @@ reearth.on("message", ({ action, payload }: PostMessageProps) => {
     });
   }
 
-  // For 3dtiles color
+  // FIXME(@keiya): We need to compose transparency with color,
+  //                but currently rgba is not working on NLS.
+  //                So I will fix NLS and implement composing color.
```

Comment:
I will fix this in other PR because this need to fix NLS.
Currently, composing transparent color with 3dtiles color is not working. For example, when I change transparency to 0.5, also change building color to blue, then the building color will be opaque.